test/environmentSettings.ts

```diff
@@ -5,14 +5,14 @@
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 
-import path from 'path';
+import { join } from 'path';
```

Comment:
Weird that `import path` doesn't work yet `import { join }` does, but if this works in CI/GHA then great.  I just pulled the branch, rebuilt, and debugged and it still worked locally.